tests/encryption_/test_management.py

```diff
@@ -8,10 +8,17 @@
 @skipUnlessDBFeature("supports_queryable_encryption")
 @modify_settings(INSTALLED_APPS={"prepend": "django_mongodb_backend"})
 class CommandTests(TestCase):
+    databases = {"default", "encrypted"}
     maxDiff = None
 
     # Expected encrypted field maps for all Encrypted* models
     expected_maps = {
+        "encryption__patientrecord": {
+            "fields": [
+                {"bsonType": "string", "path": "ssn", "queries": {"queryType": "equality"}},
+                {"bsonType": "object", "path": "billing"},
+            ]
+        },
```

Comment:
I don't think the recursive logic is correct. PatientRecord is an embedded model so it shouldn't appear in the map. Everything should be relative to Patient.

Reply:
b1e9cc08a46eee478ef55a8c980cef443fdc196b

Reply:
The logic still isn't correct [as you noted in 0d0b246a7ad7d690e922779929d4fac417dfa88a]. (Don't claim an issue fixed with no changes to tests.)

"encrypt_patientrecord" shouldn't appear in the output. It's an embedded model that doesn't have its own collection. "encrypt_patient" should be there with the full path to the embedded fields, e.g. `{"path": "patient_record.ssn"}`

Reply:
True, but pretty sure I've printed `encrypted_fields` in schema and gotten the correct schema for Patient so it may be the test that is wrong.

Reply:
I doubt the embedded fields are encrypted if the correct path isn't specified. Specifying `'path': 'ssn'` on `encryption__patient` encrypts `encryption__patient.ssn`. Even though the Django model definition doesn't have that field, there's no reason why MongoDB would raise an error.